students/Amit/Session04/get_lang.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+from collections import defaultdict
+
+def list_langs():
+
+    with open('students.txt', 'r') as file:
+        file.readline()
+        languages = [lang.capitalize() for line in file for lang in line.strip().split(':')[1].replace(",", " ").strip().split() if not lang[0].isupper()]
```

Comment:
slick! 

Though maybe better to split this out to a couple lines -- it's pretty hard to see what's going on with a line this long.